raster-foundry/raster-foundry
# Improve Scene Query Performance

## Overview

This pull request adapts scene querying to do the following:
 - Use `visibility` column to track public imagery
 - Delete public scene view access control rules in a migration
 - Update scene list query specifically to use `visibility` column and not use a JOIN
 - Add default sort ordering by acquisition date, created date, id

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

## Testing Instructions

 * Run migrations
 * Start server and browse scenes, should be sorted by acquisition date

Closes #3519, #3520, #3521
